opm/material/fluidsystems/blackoilpvt/Co2GasPvt.hpp

```diff
@@ -64,31 +68,48 @@ class Co2GasPvt
     //! The binary coefficients for brine and CO2 used by this fluid system
     using BinaryCoeffBrineCO2 = BinaryCoeff::Brine_CO2<Scalar, H2O, CO2>;
 
-    explicit Co2GasPvt() = default;
-
-    explicit Co2GasPvt(const std::vector<Scalar>& salinity,
-                       int activityModel = 3,
-                       int thermalMixingModel = 1,
-                       Scalar T_ref = 288.71, //(273.15 + 15.56)
-                       Scalar P_ref = 101325);
+    Co2GasPvt() = default;
+
+    Co2GasPvt(const ContainerT& salinity,
+              int activityModel = 3,
+              int thermalMixingModel = 1,
+              Scalar T_ref = 288.71, //(273.15 + 15.56)
+              Scalar P_ref = 101325);
+
+    Co2GasPvt(const Params& params,
+              ContainerT brineReferenceDensity,
+              ContainerT gasReferenceDensity,
+              ContainerT salinity,
+              bool enableEzrokhiDensity,
+              bool enableVaporization,
+              int activityModel,
+              Co2StoreConfig::GasMixingType gastype)
+              : brineReferenceDensity_(brineReferenceDensity)
```

Comment:
Nitpick: Better, but I think to improve the distinction between the function arguments and the init list further, the indentation level of the initialization list here should be one more than the function definition scope, i.e. starting at column 8 if the function def starts at col 4. The function arguments should retain its alignment as is.